SOURCES/sigma2aurora/sigma2aurora.py

```diff
@@ -1,22 +1,17 @@
-import xml.etree.ElementTree as ET
-import json
-import io
 import argparse
+import gzip
+import io
+import json
 import os
-import struct
 import shutil
-import gzip
-
-try:
```

Comment:
This was needed, I think on a windows system or *nix system. Don't remember it anymore.

Reply:
imports have just been re-ordered. `gzip` will be still imported. Output seems the same. I would appreciate it, if you could double check it. Thanks.

Reply:
No it looks like the the try I gone, at least in the pull request review

Reply:
Yes it is gone. It still works. I was asking myself, why would you like to use `unicode`?

You can also alter the docs here: https://github.com/freeDSP/freeDSP-aurora/wiki/Create-a-Custom-Plugin

This script runs also with python3.10. Maybe you could state this in the docs as well. Since Python 2.7 is also deprecated.